### PR TITLE
Jeoffrey Fischer --- Create SSW People Profile

### PR DESCRIPTION
**Pull request by jeoffreyfischer**
Created the SSW People Profile page with the following : 
- id
- quote
- bio
- audio

### CODE REVIEW

File: Jeoffrey-Fischer/Jeoffrey-Fischer.md
Comment (JeanThirion) [inline Jeoffrey-Fischer/Jeoffrey-Fischer.md:1]: `With a strong foundation in Aeroacoustics, he previously served as an esteemed Aeroacoustics Lecturer at UNSW for eight years`

Aeroacoustics twice - also I'm pretty sure it's more than just "strong foundation ;D"

Comment (bradystroud) [inline Jeoffrey-Fischer/Jeoffrey-Fischer.md:1]: Use markdown to bold text 
https://www.markdownguide.org/basic-syntax/#bold

File: Jeoffrey-Fischer/Jeoffrey-Fischer.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+id: "fc46b55b-98e5-ed11-8847-00224815a722"
+
+qualifications: ""
```
Comment (bradystroud) [inline Jeoffrey-Fischer/Jeoffrey-Fischer.md:4]: I would add your plane noise degree here

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: Jeoffrey-Fischer/Jeoffrey-Fischer.md
Comment (JeanThirion) [reply Jeoffrey-Fischer/Jeoffrey-Fischer.md:1]: I'd also add a line break between intro (non-tech) and the tech part

Comment (jeoffreyfischer) [reply Jeoffrey-Fischer/Jeoffrey-Fischer.md:1]: Done - thanks for that !

File: Jeoffrey-Fischer/Jeoffrey-Fischer.md (hunk 1 above)
Comment (jeoffreyfischer) [reply Jeoffrey-Fischer/Jeoffrey-Fischer.md]: Done

File: Jeoffrey-Fischer/Jeoffrey-Fischer.md
Comment (jeoffreyfischer) [reply Jeoffrey-Fischer/Jeoffrey-Fischer.md:1]: Done - sentence removed and added line breaks

File: Jeoffrey-Fischer/Jeoffrey-Fischer.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+id: "fc46b55b-98e5-ed11-8847-00224815a722"
+
+qualifications: "PhD in Aeroacoustics"
+
+quote: "Don't believe everything you read on the internet."
+quoteAuthor: "Abraham Lincoln"
+---
+
+[Editing your profile]: https://github.com/SSWConsulting/People/wiki/3.-Editing-your-profile
```
Comment (bradystroud) [inline Jeoffrey-Fischer/Jeoffrey-Fischer.md:10]: remove this

Comment (jeoffreyfischer) [reply Jeoffrey-Fischer/Jeoffrey-Fischer.md]: Done

**Review by bradystroud — APPROVED** [review]

**Pull request merged**